tests/testthat/test-independent-testing-meta_adam.R

```diff
@@ -0,0 +1,25 @@
+library(r2rtf)
+
+
+test_that("meta_adam testing", {
+  test <- meta_adam(
+    population = r2rtf::r2rtf_adsl,
+    observation = r2rtf::r2rtf_adae
+  )
+  expect_equal(class(test), "meta_adam")
+  expect_equal(test$plan, list())
+  expect_equal(test$population, list())
+  expect_equal(test$observation, list())
+  expect_equal(test$parameter, list())
+  expect_equal(test$analysis, list())
+  
+  meta <- meta_dummy()
```

Comment:
Do we need this (line 16 - 21)?

Reply:
> Do we need this (line 16 - 21)?

Yes, in order to test "The plan, observation, population, parameters, and analysis will be reported if one further define them into the metadata"

Reply:
But the `meta_adam` source code is just for initialization purposes. Plans and parameters are passed in later when you use `meta_build`. Please correct me if I misunderstand the code. I also updated the test plan of `meta_adam` and removed "The plan, observation, population, parameters, and analysis will be reported if one further defines them into the metadata".